guardian/newswires
# Initial set up for a wires UI/API server

## What does this change?

Basic hello world service, with auth and UI and CI and infra

## How to test

Deploy to CODE and see if you can get to a UI

https://newswires.code.dev-gutools.co.uk
